js/foundation.tabs.js

```diff
@@ -65,7 +65,6 @@ class Tabs {
 
       $tabContent.attr({
         'role': 'tabpanel',
-        'aria-hidden': !isActive,
```

Comment:
Don't we still need to set aria-hidden in the case that we are hidden? If I'm reading the issue properly, the behavior is that we should still have aria-hidden when hidden, but have no attribute rather than a "false" when not hidden.

I think we probably need to use attr in the hidden case and removeAttr in the unhidden case, yes?